runner/scripts/4-analyzeAssignmentOutput.ts

```diff
@@ -0,0 +1 @@
+// ToDo
```

Comment:
What would analyze be for?

Reply:
This would run the scraper output from step 3 assignments against the probes generated in step 2, which would use the full analyze code/plugins surface area.